pkg/pgmodel/metrics/ingest.go

```diff
@@ -151,6 +151,14 @@ var (
 			Help:      "Total items (samples/exemplars/spans) received.",
 		}, []string{"type", "kind"},
 	)
+	IngestorBytes = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: util.PromNamespace,
+			Subsystem: "ingest",
+			Name:      "bytes_total",
```

Comment:
```suggestion
			Name:      "requests_bytes_total",
```

nit: we can add `requests_` to reduce potential confusion in what those bytes represent. Especially since some folks could potentially deduct that `Total bytes ingested` translates to "current database size".

Reply:
Makes sense